CreeDictionary/CreeDictionary/management/commands/convertndstopanes.py

```diff
@@ -106,33 +107,40 @@ def convert_layout_to_new_format(self, layout):
                 cells.append(new_cell)
                 first_cell = False
 
-            rows.append(Row(cells))
+            rows.append(ContentRow(cells))
 
         return ParadigmTemplate(Pane(rows) for rows in panes)
 
 
-class UnknownLabelTagMixin(BaseLabelCell):
+class UnknownLabelTagMixin:
     """
     Mixin to a RowLabel or ColumnLabel to display <!ORIGINAL LABEL!> instead of an FST tag.
     """
 
     UNANALYZABLE = ("?",)
 
-    def __init__(self, original: str):
+    def __init__(self, original):
```

Comment:
Is there any particular reason the type of `original` was removed?

Reply:
Haha, this is exactly the kind of code review comment I hope to get! I removed it for sloppy coding reasons. Basically, removing all type annotations make the function no longer analyzable by `mypy`. Because I removed `BaseLabelCell` from the mixin `mypy` can no longer verify that `super().__init__(self.UNANALYZABLE)` is a type-safe call.

I'll research how to make this statically type-safe! 😅